### PR TITLE
packaging for release 4.11.0

### PR DESCRIPTION
**Pull request by jtgrenz**
Version release with the new inventory level apis 

@celsodantas

### CODE REVIEW

**Comment by jamiemtdwyer** [comment]
🙌

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,9 @@
+== Version 4.11.0
+
+* Added `ShopifyAPI::InventoryItem`
+* Added `ShopifyAPI::InventoryItem`
```
Comment (celsodantas) [inline CHANGELOG]: duplicate `ShopifyAPI::InventoryItem` 😬

Comment (jtgrenz) [reply CHANGELOG]: 🤦‍♂️

Comment (jamiemtdwyer) [reply CHANGELOG]: Whoops 😅

**Pull request merged**